ocaml/parsing/ast_invariants.ml

```diff
@@ -39,18 +39,9 @@ let module_type_substitution_missing_rhs loc =
   err loc "Module type substitution with no right hand side"
 let empty_comprehension loc = err loc "Comprehension with no clauses"
 let function_without_value_parameters loc =
-<<<<<<< HEAD
-  err loc "Function without any value parameters"
-||||||| a198127529
-let no_val_params loc = err loc "Functions must have a value parameter."
-
-let non_jane_syntax_function loc =
-  err loc "Functions must be constructed using Jane Street syntax."
-=======
   err loc "Functions must have a value parameter."
 let redundant_nested_constraints loc =
   err loc "Nested pattern constraints must all specify a type"
->>>>>>> flambda-backend/main
```

Comment:
I suggest keeping the `err loc "Function without any value parameters"` portion from HEAD to minimize diff with upstream.